lib/ukrust/Config.uk

```diff
@@ -0,0 +1,9 @@
+config LIBUKRUST
+	bool "ukrust: Rust support for Unikraft's internals"
+	depends on ARCH_X86_64
+	default n
+	help
+	  This library provides all the necessary functionalities
+	  for writing Rust internal libraries. It exports Rust
```

Comment:
How about this change `for writing Rust internal libraries` --> `for Rust libraries that utilize Unikraft's native API`? In our terminology, an internal library is directly located in the Unikraft source tree and I think you use this also for external libraries that are written in Rust and utilize Unikraft APIs.